### ISSUE TITLE
vq-wav2vec train.py unrecognized arguments

### ISSUE THREAD

**Issue by Lauren-Eyler** [pytorch/fairseq#2051]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by Lauren-Eyler** [comment]
Hello, I am hoping to train a new vq-wav2vec model as described in the example on this page:https://github.com/pytorch/fairseq/tree/master/examples/wav2vec. I am using Google Colab with the code from this page, edited to include the paths for my files and manifest:

! python train.py /manifest/path --save-dir /model/path --num-workers 6 --fp16 --max-update 400000 \
--save-interval 1 --no-epoch-checkpoints --arch wav2vec --task audio_pretraining --lr 1e-06 --min-lr 1e-09 \
--optimizer adam --max-lr 1e-05 --lr-scheduler cosine \
--conv-feature-layers "[(512, 10, 5), (512, 8, 4), (512, 4, 2), (512, 4, 2), (512, 4, 2), (512, 1, 1), (512, 1, 1), (512, 1, 1)]" \
--conv-aggregator-layers "[(512, 2, 1), (512, 3, 1), (512, 4, 1), (512, 5, 1), (512, 6, 1), (512, 7, 1), (512, 8, 1), (512, 9, 1), (512, 10, 1), (512, 11, 1), (512, 12, 1), (512, 13, 1)]" \
--activation gelu --offset auto --skip-connections-agg --residual-scale 0.5 \
--log-keys ["prob_perplexity","code_perplexity","temp"] --vq-type gumbel --vq-groups 2 --vq-depth 2 \
--combine-groups --vq-vars 320 --vq-temp "(2,0.5,0.999995)" --prediction-steps 12 --warmup-updates 1000 \
--warmup-init-lr 1e-07 --criterion binary_cross_entropy --num-negatives 10 --max-sample-size 150000 \
--max-tokens 300000 --cross-sample-negatives 0 --update-freq 1 --seed 2 --skip-invalid-size-inputs-valid-test

I get the following error: train.py: error: unrecognized arguments:  --log-keys [prob_perplexity,code_perplexity,temp]

Do you know how to fix this problem? I appologize if I am missing something obvious as I am new to Python. Thank you!

**Comment by lematt1991** [comment]
What version of fairseq are you using?  It looks like 3335de5f441ee1b3824e16dcd98db620e40beaba was the commit that added `--log-keys`, so you will need to use `master`, not the latest release.  You should be able to do `pip install git+https://github.com/pytorch/fairseq` to install the latest master version.
